### ISSUE TITLE
NullPointerException for invalid ttl file

### ISSUE THREAD

**Issue by JJ-Author** [imas/rdflint#116]
Hi, maybe I am still confused what this linter is supposed to do, but I thought it would show me errors in ttl files. I just inserted an invalid ttl file to test and as a result I get a NullPointerException and there seems no way to actually find the incorrect ttl file in the log. 

https://github.com/dbpedia/format-mappings/runs/554543342?check_suite_focus=true

```
Run rdflint
  rdflint
  shell: /bin/bash -e {0}
  env:
    JAVA_HOME: /opt/hostedtoolcache/jdk/11.0.6/x64
    JAVA_HOME_11.0.6_x64: /opt/hostedtoolcache/jdk/11.0.6/x64
Exception in thread "main" java.lang.NullPointerException
	at org.apache.jena.riot.system.RiotLib.isBNodeIRI(RiotLib.java:86)
	at org.apache.jena.riot.system.ParserProfileStd.createURI(ParserProfileStd.java:152)
	at org.apache.jena.riot.system.ParserProfileStd.create(ParserProfileStd.java:216)
	at org.apache.jena.riot.lang.LangTurtleBase.tokenAsNode(LangTurtleBase.java:498)
	at org.apache.jena.riot.lang.LangTurtleBase.node(LangTurtleBase.java:343)
	at org.apache.jena.riot.lang.LangTurtleBase.triples(LangTurtleBase.java:245)
	at org.apache.jena.riot.lang.LangTurtleBase.triplesSameSubject(LangTurtleBase.java:191)
	at org.apache.jena.riot.lang.LangTurtle.oneTopLevelElement(LangTurtle.java:46)
	at org.apache.jena.riot.lang.LangTurtleBase.runParser(LangTurtleBase.java:91)
	at org.apache.jena.riot.lang.LangBase.parse(LangBase.java:41)
	at org.apache.jena.riot.RDFParserRegistry$ReaderRIOTLang.read(RDFParserRegistry.java:191)
	at org.apache.jena.riot.RDFParser.read(RDFParser.java:352)
	at org.apache.jena.riot.RDFParser.parseURI(RDFParser.java:321)
	at org.apache.jena.riot.RDFParser.parse(RDFParser.java:295)
	at org.apache.jena.riot.RDFParser.parse(RDFParser.java:237)
	at com.github.imas.rdflint.validator.impl.RdfSyntaxValidator.validateFile(RdfSyntaxValidator.java:65)
	at com.github.imas.rdflint.ValidationRunner.lambda$null$4(ValidationRunner.java:77)
	at java.base/java.lang.Iterable.forEach(Iterable.java:75)
	at com.github.imas.rdflint.ValidationRunner.lambda$execute$5(ValidationRunner.java:77)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)

```

**Comment by JJ-Author** [comment]
you are right, i had a look into the typescript, it only installs it to the path. so the problem seems on rdflint side. thanks for transferring the issue.

**Comment by takemikami** [comment]
fixed, at v0.1.4
